src/components/folderToolBar/FolderToolBar.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import * as S from "./FolderToolBar.style";
+import { FOLDER_MANAGE_MENUS } from "../../constants/constants";
+
+const FolderToolbar = ({ folderName, setModalComponent }) => {
+  const handleMenuClick = (e, menu) => {
+    e.stopPropagation();
+    const { Modal, modalTitle, buttonText } = menu;
+    setModalComponent(<Modal modalTitle={modalTitle} buttonText={buttonText} modalTarget={folderName} />);
+  };
+
+  return (
+    <S.ToolBarWrap>
+      <S.FolderName>{folderName || "전체"}</S.FolderName>
+      <S.ManageButtonWrap>
+        {folderName &&
+          FOLDER_MANAGE_MENUS.map((menu, i) => (
+            <li key={menu.iconTitle}>
```

Comment:
React 팀에서는 key의 값으로 index 자체만 넣은 것을 피하라고 말하고 있어 다른 값을 넣어주신 것은 좋습니다 👍
다만 key에 어떤 조합을 통해 unique key를 최대한 보장하도록 해주시면 더 좋을 것 같습니다.
```suggestion
            <li key={menu.iconTitle + '- ' + i }>
```